code/yarn.lock

```diff
@@ -10221,6 +10221,8 @@ react-router@6.27.0:
   dependencies:
     "@remix-run/router" "1.20.0"
 
+<<<<<<< HEAD
```

Comment:
Resolve this conflict. Open this file, scroll to this line and **Accept Incoming Change**